app/server/toronto-bids/toronto-bids/src/toronto_bids/core.clj

```diff
@@ -8,38 +8,77 @@
 	 (:require [clojure.java.jdbc :as jdbc])
 	 (:require [clojure.data.json :as json])
 	 (:require [toronto-bids.stuff :as stuff])
+	 (:require [toronto-bids.documents :as documents])
 ;	 (:require [ring-debug-logging.core :as debug])
 )
 
-(defn write-simple [table columns]
+(defn write-simple [db table columns]
```

Comment:
Admittedly this is pedantic but could we rename this function to something like 'read-db-columns' to reflect the fact that nothing is being written to the db?

Reply:
The write is referring to the output stream.  Could change all the write names to output.  Would that be better?